synapse/storage/databases/main/room.py

```diff
@@ -1596,6 +1597,8 @@ async def get_event_reports_paginate(
                 oldest first (forwards)
             user_id: search for user_id. Ignored if user_id is None
             room_id: search for room_id. Ignored if room_id is None
+                event_sender_user_id: search for the sender of the reported event. Ignored if
+            event_sender_user_id is None
```

Comment:
Previous conversation: https://github.com/element-hq/synapse/pull/17948#discussion_r1868340282

```suggestion
            event_sender_user_id: search for the sender of the reported event. Ignored if
                event_sender_user_id is None
```